WordPressComKitTests/MeServiceTests.swift

```diff
@@ -0,0 +1,55 @@
+import XCTest
+import OHHTTPStubs
+import WordPressComKit
+
+class MeServiceTests: XCTestCase {
+    var subject: MeService!
+    
+    override func setUp() {
+        super.setUp()
+        
+        subject = MeService()
+    }
+    
+    override func tearDown() {
+        super.tearDown()
+        
+        subject = nil
+        OHHTTPStubs.removeAllStubs()
+    }
+    
+    func testFetchMe() {
+        stub(isMethodGET() && isHost("public-api.wordpress.com") && isPath("/rest/v1.1/me")) { _ in
```

Comment:
I think we should structure the code in a way that you don't need stubs. We should test:
- Request serialization: does `fetchMe` results in a `NSURLRequest` to `/rest/v1.1/me` with no parameters?
- Response parsing: does `MeService.mapJSONToMe()` turn `me.json` into the expected `Me` object?

In a way, that's actually what the library does, and it could also mean that we don't need to require Alamofire, we just offer a way to generate requests and parse responses. We could provide a default networking implementation on a subspec, or leave the networking to the calling app. Random idea though, I'm not sure how useful that'd be or if it would complicate things, but just something to think about.